src/apps/components/AppDialog/AppDialog.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  Dialog,
+  DialogContent,
+  DialogProps,
+  DialogTitle,
+  IconButton,
+  Typography
+} from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+import React from "react";
+
+import { useStyles } from "./styles";
+
+interface AppDialogProps extends DialogProps {
+  onClose(): void;
```

Comment:
Let's be consistent with the rest of the project

```suggestion
  onClose: () => void;
```